apps/storefront/src/pages/Login/index.tsx

```diff
@@ -148,76 +169,42 @@ export default function Login(props: PageProps) {
           return;
         }
 
-        setLoginInfo(Info);
         setLoading(false);
-      } catch (e) {
-        setLoginInfo(initialLoginInfo);
       } finally {
         setLoading(false);
       }
     };
 
     init();
-    // disabling as we only need to run this on the first render and its causing infinite loops when loging in
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [b3Lang, endMasquerade, isLoggedIn, isMasquerade, searchParams]);
```

Comment:
This doesn't seem to cause infinite loops anymore:

https://github.com/bigcommerce/b2b-buyer-portal/assets/4542735/e7868291-779e-4b24-98e1-f590877aeb08